test/jdk/java/lang/String/TranslateEscapes.java

```diff
@@ -94,6 +110,22 @@ static void verifyEscape(String string, char ch) {
         }
     }
 
+    static void verifyEscape(String string1, String string2) {
```

Comment:
These are unicode escapes too.  The method name should reflect that.

Reply:
Changed